example/conanfile.py

```
@@ -9,12 +11,48 @@ class MothGraphicsExample(ConanFile):
     generators = "CMakeToolchain", "CMakeDeps"
     package_type = "application"
 
+    options = {
+        "disable_vulkan": [True, False],
+        "disable_sdl": [True, False],
+    }
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)